hazelcast/src/main/java/com/hazelcast/core/IAtomicLong.java

```diff
@@ -40,6 +40,15 @@
  * </pre>
  * </p>
  *
+ * During a network partition event it is possible for the register to exist in each of the partitioned clusters or to
```

Comment:
Minor comment : I would prefer sticking to `IAtomicLong` or `instance` rather than introducing `register` as a new term in the javadoc.